bob/src/cleaner.rs

```diff
@@ -25,14 +28,26 @@ impl Cleaner {
             self.old_blobs_check_timeout,
             self.soft_open_blobs,
             self.hard_open_blobs,
+            self.filter_memory_limit,
         ));
     }
 
-    async fn task(backend: Arc<Backend>, t: Duration, soft: usize, hard: usize) {
+    async fn task(
+        backend: Arc<Backend>,
+        t: Duration,
+        soft: usize,
+        hard: usize,
+        filter_memory_limit: Option<usize>,
+    ) {
         let mut interval = interval(t);
         loop {
             interval.tick().await;
             backend.close_unneeded_active_blobs(soft, hard).await;
+            if let Some(limit) = filter_memory_limit {
+                log::warn!("Memory before: {}", backend.filter_memory_allocated().await);
```

Comment:
IMO it should be `info`, but it's debatable

Reply:
+1